signature/policy_reference_match_test.go

```diff
@@ -358,3 +412,193 @@ func TestPRMExactRepositoryMatchesDockerReference(t *testing.T) {
 		testExactPRMAndSig(t, prmExactRepositoryFactory, test.refB, test.refA, test.result)
 	}
 }
+
+func TestPRMRemapIdentityRefMatchesPrefix(t *testing.T) {
+	for _, c := range []struct {
+		ref, prefix string
+		expected    bool
+	}{
+		// Prefix is a reference.Domain() value
+		{"docker.io/image", "docker.io", true},
+		{"docker.io/image", "example.com", false},
+		{"example.com:5000/image", "example.com:5000", true},
+		{"example.com:50000/image", "example.com:5000", false},
+		{"example.com:5000/image", "example.com", false},
```

Comment:
This case may be worth mentioning explicitly in the docs.  Some users may assume that "host" would prefix "host:port".

Reply:
Added, as well as a note that references not matching `prefix` are not remapped, and a heading to make allow faster skimming.

(The man page formatting is not good, but it is at least not worse than before.)

Reply:
I don't see a new commit? Did you push?

Reply:
Oops, pushed the wrong repo. Updated now.